### ISSUE TITLE
Scielo ISSN 0718-9273 The living body and transcendental subjectivity in the phenomenology of Edmund Husserl - 773

### ISSUE THREAD

**Issue by kdr154** [ubtue/DatenProbleme#63]
773 ist unvollständig. $g fehlt

**Comment by kdr154** [comment]
Nein, hier fehlt $g weiterhin in 773. 936 ist besser, aber auch noch nicht richtig. Die 40 muss in $d und nicht, wie hier weiterhin, in $e

**Comment by kdr154** [comment]
Möglicherweise. Sieht mir ganz so aus wie das Problem mit ZS die von Dialnet gehostet werden. Dort kamen oder kommen alle Bandangaben, auf die mit No, Numero ... referiert werden ins Zotero-Feld Ausgabe, das ist das Feld für die Heftzählung. Offensichtlich wird der Ausdruck Nummer, No, Numero ... als Heftzählung interpretiert.

**Comment by mtrojan-ub** [comment]
... und in diesem Fall wäre es dann ja OK dass 773g fehlt, weil es sich bei der 40 um die Heftnummer handelt und die Bandnummer fehlt, siehe #62.
Passt das so?
